SharpPcap/LibPcap/PcapHeader.cs

```diff
@@ -132,6 +132,14 @@ public PcapHeader(uint seconds, uint microseconds, uint packetLength, uint captu
         /// </summary>
         public uint CaptureLength;
 
+        public PosixTimeval Timeval
+        {
+            get
+            {
+                return new PosixTimeval(Seconds, MicroSeconds);
+            }
+        }
```

Comment:
What would happen if user do `header.Timeval.Seconds = 5` ?

Reply:
they'd change the new PosixTimeval instance? how should we handle this? we could somehow let the user change the header seconds/microseconds but nothing comes to mind. What did you have in mind?

Reply:
store the timeval in a field

Reply:
right, so in that case a change would persist but still wouldn't be particularly useful would it? should we flag the returned object as readonly to prevent it from being altered? Nothing so far is using the value.

Reply:
@kayoub5 if we do add a private field wouldn't we end up potentially altering the marshaling of PcapHeader? Thoughts?

Reply:
true..., we could ignore this until someone needs it / request it.